translations/validators.fr.yml

```diff
@@ -29,6 +29,9 @@ common.email.repeated: Les adresses email ne correspondent pas.
 common.country.not_blank: Cette valeur ne doit pas être vide.
 common.country.invalid: Ce pays n'est pas valide.
 
+common.nationality.not_blank: Cette valeur ne doit pas être vide.
```

Comment:
Why do you need to custom the default NotBlank constrain's message?

Reply:
All NotBlank's messages are override on object MembershipRequest, I think it's for the translation